lib/http/service.js

```diff
@@ -58,6 +58,17 @@ module.exports = (container) => {
   const cookieParser = require('cookie-parser');
   service.use(cookieParser());
 
+  service.use((req, res, next) => {
+    // TODO only set for safe methods?
+    // TODO only set for authenticated requests?
```

Comment:
Are these todos for a reviewer to consider? Something you're still thinking about?

Reply:
Good question.  I think I'd be happy deleting them both, but I'd be interested in a review of that.

Reply:
I've removed the TODOs - it seems reasonable to be more aggressive with cache-busting.